### DISCUSSION TITLE
How does one disable the many DEB | [AssetHandler] logs?

### DISCUSSION THREAD

**Discussion by danawoodman** [wailsapp/wails#3177] [Q&A]
I've attempted to set `LogLevel` to `logger.ERROR` to disable these logs but not luck. 

Did a bunch of googling / reading the docs as well and see no way to disable these asset logs which are basically just a ton of Vite logs I dont' care about but make me lose all useful logs.

![main go — cnc 2024-01-06 at 6 30 11 PM](https://github.com/wailsapp/wails/assets/157695/6123dd81-f43d-4cc8-a24e-84c6af3f51b4)

Wails version: 

```
github.com/wailsapp/wails/v2 v2.7.1
```

**Comment by Sammy-T** [comment]
Setting the `LogLevel` option didn't appear to do anything for me either. 

But running the [dev](https://wails.io/docs/reference/cli#dev) command with the `loglevel` flag set seems to work:

```bash
wails dev -loglevel Error
```
